db-migrations/1_create_schema.up.sql

```diff
@@ -3,7 +3,8 @@ DROP SCHEMA IF EXISTS surveyv2 CASCADE;
 CREATE SCHEMA surveyv2;
 
 CREATE TABLE IF NOT EXISTS surveyv2.survey (
-    survey_ref uuid PRIMARY KEY,
+    id uuid PRIMARY KEY,
+    survey_ref smallint,
```

Comment:
This should be string so that we can store survey_ref with a leading zero, eg 039